src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,16 +1,58 @@
 package wolox.training.controllers;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.server.ResponseStatusException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
 @Controller
```

Comment:
Maybe in the future, when UI is no merged with business logic, you can use `@RestController` for a REST API